scripts/cli.py

```
@@ -0,0 +1,407 @@
+        
+        deployment = AutomatedDeployment(version, environment)
+        
+        if auto_approve:
+            click.echo(f"🚀 Starting automated deployment of {version} to {environment}")
+        else:
+            click.echo(f"📋 Creating deployment plan for {version} to {environment}")
+        
+        success = deployment.run()
+        
+        if success:
+            click.echo(f"✅ Deployment completed successfully")
```

Ruff (0.11.9)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)